constraint/Makefile

```diff
@@ -33,8 +40,18 @@ vet:
 	go vet ./pkg/...
 
 # Generate code
-generate:
+generate: .output/buildutil
 ifndef GOPATH
 	$(error GOPATH not defined, please define GOPATH. Run "go help gopath" to learn more about GOPATH)
 endif
-	go generate ./pkg/...
+	export GOROOT=$(go env root)
+	GO111MODULE=off go generate ./pkg/...
+
+# find or download controller-gen
+# download controller-gen if necessary
+.output/buildutil:
+	mkdir -p .output
+	docker build buildutils -t maketools && docker run -v ${PWD}/.output:/.output maketools
+	echo "buildutils generated:"
+	ls -l .output/buildutil
```

Comment:
this is listed twice here and in maketools

Reply:
Moot now (removed).